test/parallel/test-vm-basic.js

```diff
@@ -69,3 +69,56 @@ assert.strictEqual(result, 'undefined');
 const sandbox3 = {};
 const context2 = vm.createContext(sandbox3);
 assert.strictEqual(sandbox3, context2);
+
+// Test 6: invalid arguments
```

Comment:
I wouldn't be opposed to a follow up PR that removes these "Test n:" comments.

Reply:
I just went with what was already there 😅